op-chain-ops/cmd/check-l2/main.go

```diff
@@ -429,6 +440,25 @@ func checkL2ERC721Bridge(addr common.Address, client *ethclient.Client) error {
 		return err
 	}
 	log.Info("L2ERC721Bridge", "_initialized", initialized)
+	if initialized.Uint64() != genesis.InitializedValue {
+		return fmt.Errorf("%w: %s", errInvalidInitialized, initialized)
+	}
+
+	abi, err := bindings.L2ERC721BridgeMetaData.GetAbi()
+	if err != nil {
+		return err
+	}
+	calldata, err := abi.Pack("initialize")
+	if err != nil {
+		return err
+	}
+	msg := ethereum.CallMsg{
+		To:   &addr,
+		Data: calldata,
+	}
+	if _, err := client.CallContract(context.Background(), msg, nil); err.Error() != errAlreadyInitialized.Error() {
```

Comment:
A bit brittle relying on error strings here as we upgrade go-ethereum. If we're sure that this error string format is something go-ethereum will continue to maintain this error format then disregard this comment.
But I'd be more comfortable with this if we had a Negative Test that asserts the error is returned when an invalid initialized value is used.

Reply:
We will soon have solidity test coverage of this that will reduce this from being a critical test

Reply:
I updated the logic to be more lenient to changes to geth by using string matching instead of an exact comparison